script.lazytv/service.py

```diff
@@ -0,0 +1,1649 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#  Copyright (C) 2013 KodeKarnage
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with XBMC; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+
+"""
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#@@@@@@@@@@
+#@@@@@@@@@@ - allow for next ep notification in LazyTV smartplaylist READY FOR TESTING
+#@@@@@@@@@@ - suppress notification at start up READY FOR TESTING
+#@@@@@@@@@@ - improve handling of specials
+#@@@@@@@@@@ - improve refreshing of LazyTV Show Me window
+#@@@@@@@@@@
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"""
+
+
+import xbmc
```

Comment:
same as the script entrypoint. Usually a good ideia to make it as small as possible.